src/utils/nostr.ts

```diff
@@ -24,8 +26,11 @@ const defaultRelays = [
    'wss://relay.nos.social',
 ];
 
-const initializeNDK = async () => {
-   let privkey = process.env.BOARDWALK_NOSTR_PRIVKEY;
+const initializeNDK = async (relays = defaultRelays) => {
+   console.log('initializing NDK with relays', relays);
+   let privkey =
+      process.env.NEXT_PUBLIC_BOARDWALK_NOSTR_PRIVKEY ||
+      '9baeb030d6734ccd2e1da217b2e0e8e4033389b93548549cccf63c12baefbd70';
```

Comment:
It was a quick hack because I couldn't figure out why the key wasn't being read from the vercel env.

I fixed it now. Its a bit weird because these utils are used server side and on the frontend.